src/main/kotlin/lotto/Statistics.kt

```diff
@@ -8,11 +8,18 @@ class Statistics(winningLotto: WinningLotto, lottos: List<Lotto>) {
     private val totalMoney = lottos.size * LottoSeller.LOTTO_PRICE
 
     init {
-        lottos.map { lottoRanks.add(LottoRank.selectRank(winningLotto.matchCount(it.numbers))) }
+        lottos.map {
+            lottoRanks.add(
+                LottoRank.selectRank(
+                    winningLotto.matchCount(it.lotto),
+                    winningLotto.matchBonus(it.lotto)
+                )
+            )
+        }
     }
 
     fun getWinningResult(): Map<LottoRank, Int> {
-        return lottoRanks.getRanks()
+        return lottoRanks.getRanks().filterNot { it.key == LottoRank.LOSE }
```

Comment:
```suggestion
        return lottoRanks.getRanks()
            .filterNot { it.key == LottoRank.LOSE }
```
함수 체이닝 컨벤션을 적용하면 좋을 것 같아요.